src/store/reducers/heatmap.ts

```diff
@@ -47,16 +57,16 @@ const tablets = function z(state = initialState, action) {
     }
 };
 
-export function getTabletsInfo({nodes, path}) {
+export function getTabletsInfo({nodes, path}: IHeatmapApiRequestParams) {
     return createApiRequest({
         request: Promise.all([
             window.api.getTabletsInfo({nodes, path}),
             window.api.getHeatmapData({path}),
         ]),
-        actions: FETCH_TABLETS,
-        dataHandler: ([tabletsData = [], describe = {}]) => {
+        actions: FETCH_HEATMAP,
+        dataHandler: ([tabletsData = {}, describe = {}]) => {
             const {TabletStateInfo: tablets = []} = tabletsData;
-            const TabletsMap = new Map();
+            const TabletsMap: Map<string | undefined, IHeatmapTabletData> = new Map();
```

Comment:
I'd say it's better to explicitly skip empty ids, and keep the keys type consistent as a string